section4/exercises/dog.rb

```diff
@@ -13,18 +14,24 @@ def initialize(breed, name, age)
   end
 
   def bark
-    p "woof!"
+    puts "woof!"
   end
 
   def eat
+    puts "Is the dog hungry?"
     @hungry = false
   end
+
+  def play
+    puts "The dog plays all day long and is it hungry now?"
+    @hungry = true
+  end
 end
 
 fido = Dog.new("Bernese", "Fido", 4)
-p fido.breed
-p fido.name
-p fido.age
-p fido.hungry
-fido.eat
-p fido.hungry
+puts fido.breed
+puts fido.name
+puts fido.age
+puts fido.hungry
+puts fido.eat
+puts fido.play
```

Comment:
Make sure you are reading instructions carefully - all that is missing is printing the dogs hunger level after playing